index.html

```diff
@@ -59,134 +72,148 @@ <h2 class="connect-me">LET'S CONNECT</h2>
       </ul>
     </section>
 
-    <section class="projects">
+    <section class="projects" id="portfolio">
       <ul class="project-list">
         <li class="project">
-          <img src="./img/project1.png" alt="" />
-          <h2 class="project-title">Tonic</h2>
+          <img src="./img/project0.png" alt="" />
+          <div class="project-info">
+                <h2 class="project-title">Tonic</h2>
 
-          <div class="stats">
-            <span class="company">CANOPY</span>
-            <span class="role-and-year">Back End Dev</span>
-            <span class="role-and-year">2015</span>
-          </div>
+                <div class="stats">
+                    <span class="company">CANOPY</span>
+                    <span class="role-and-year">Back End Dev</span>
+                    <span class="role-and-year">2015</span>
+                </div>
 
-          <p class="description">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
```

Comment:
- [ ] Kindly adjust the image and project-info to fit in the required change. 
- The image should occupy half of the card, increase the height and width.
- The project-info should be properly aligned horizontally. The `role-and-year` style is also affecting the required design for that section.
_**Refer to my comments above about the design.**_ 
_See the images below for clarity._

**Original** | **Your clone**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/121506976-15ed8900-c999-11eb-9e73-8c9a648bf07b.png) | ![image](https://user-images.githubusercontent.com/49621279/121507538-957b5800-c999-11eb-96be-f1051e884bdd.png)